src/test/java/com/amazon/project/demo/service/impl/FileReaderServiceImplTest.java

```diff
@@ -0,0 +1,32 @@
+package com.amazon.project.demo.service.impl;
+
+import com.amazon.project.demo.service.FileReaderService;
+import org.junit.Assert;
+import org.junit.Test;
+import java.util.List;
+
+public class FileReaderServiceImplTest {
+    private static final String TEST_FILE = "src/main/resources/testFile.csv";
```

Comment:
```suggestion
    private static final String TEST_FILE = "src/test/resources/testFile.csv";
```
resources for tests should better lay in `src/main/resources/`